test_haystack/mocks.py

```diff
@@ -2,7 +2,8 @@
 
 from __future__ import absolute_import, division, print_function, unicode_literals
 
-from django.db.models.loading import get_model
+from django.apps import apps
+get_model = apps.get_model
```

Comment:
This could be changed to use `from haystack.utils.app_loading import haystack_get_model` which exists for this reason.